platform/server/detect.py

```diff
@@ -107,23 +107,27 @@ def configure(env):
         env.Append(LINKFLAGS=["-ftest-coverage", "-fprofile-arcs"])
 
     if env["use_ubsan"] or env["use_asan"] or env["use_lsan"] or env["use_tsan"]:
-        env.extra_suffix += "s"
+        env.extra_suffix += "."
```

Comment:
I think this `"."` could help a lot visually, currently compiling with `extra_suffix` and sanitizer support: #37301.